src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -771,6 +771,17 @@
         --resource-group MyResourceGroup --remove recommended.vCpUs.min
 """
 
+helps['sig image-definition list-community'] = """
+type: command
+short-summary: List VM Image definitions in a gallery community (preview).
+long-summary: List VM Image definitions in a gallery community (private preview feature, please contact community image gallery team by email sigpmdev@microsoft.com to register for preview if you're interested in using this feature).
```

Comment:
Remove `(private preview feature, please contact community image gallery team by email sigpmdev@microsoft.com to register for preview if you're interested in using this feature).`